ci-operator/config/coreos/rpm-ostree/coreos-rpm-ostree-master.yaml

```diff
@@ -0,0 +1,28 @@
+build_root:
+  image_stream_tag:
+    cluster: https://api.ci.openshift.org
```

Comment:
Remove `cluster: `, it's never needed and causes problems.

Reply:
Done!